src/main/java/org/zone/region/flag/entity/player/move/message/greetings/GreetingsFlagType.java

```diff
@@ -30,12 +34,25 @@
 
     @Override
     public @NotNull GreetingsFlag load(@NotNull ConfigurationNode node) throws IOException {
-        String message = node.node("Message").getString();
+        String message = node.node("GreetingsMessage").getString();
```

Comment:
For backwards compatbility with 1.0.0, you must maintain the node that came before

Reply:
sad